### PR TITLE
Improve help of ssh

### PR DESCRIPTION
**Pull request by miya0001**
I am not sure that is my pull-request better or not. 
But I feel we should improve help for `--ssh`.

```
$ wp help
  ...
  --ssh=<user>[:<pass>]@<host>[:<port>]<path>
      Perform operation against a remote server over SSH.
```

### CODE REVIEW

File: php/config-spec.php [hunk 1]
```diff
@@ -8,8 +8,8 @@
 	),
 
 	'ssh' => array(
-		'runtime' => '=<ssh>',
-		'file' => '<ssh>',
+		'runtime' => '=<user>[:<pass>]@<host>[:<port>]<path>',
+		'file' => '<user>[:<pass>]@<host>[:<port>]<path>',
```
Comment (danielbachhuber) [inline php/config-spec.php]: `<user>` is optional, as is `[<path>]`. Really, only `<host>` is required.

Is `[:<pass>]` actually supported?

Comment (miya0001) [inline php/config-spec.php]: > Is [:<pass>] actually supported?

Oh ... orz

File: php/config-spec.php [hunk 2]
```diff
@@ -8,8 +8,8 @@
 	),
 
 	'ssh' => array(
-		'runtime' => '=<ssh>',
-		'file' => '<ssh>',
+		'runtime' => '=[<user>]@<host>[:<port>][<path>]',
+		'file' => '[<user>]@<host>[:<port>][<path>]',
```
Comment (danielbachhuber) [inline php/config-spec.php]: This should be `[<user>@]`, because the `@` isn't necessary when the user isn't included.

Comment (miya0001) [inline php/config-spec.php]: Thanks!

**Pull request merged**